ci-operator/config/openshift/osde2e/openshift-osde2e-main.yaml

```diff
@@ -53,6 +53,28 @@ tests:
   - mount_path: /usr/local/sd-cicd-aws-prod
     name: sd-cicd-aws-prod
   skip_if_only_changed: ^(?:docs|\.github)|\.md$|^(?:\.gitignore|.golang-ci.yml|OWNERS|LICENSE)$
+- as: rosa-stage-mcvw
+  run_if_changed: pkg/e2e/webhooks
```

Comment:
@jbpratt We probably will want to modify the `verify` job config to append to the `skip_if_only_changed` to ignore webhook tests. What do you think? This way if a PR is submitted for webhook test changes, it will not try to run both these pre-submit jobs.